src/pages/index.js

```diff
@@ -74,25 +69,23 @@ Promise.all([api.getInitialCards(), api.getUserInfo()])
      console.log("Form Data:", formData)
     userinfo.setUserInfo(formData);
     userinfo.setAvatar(formData.avatar);
-    cardData.forEach((item) => {
+    section.renderItems(cardData.forEach((item) => {
+      console.log("is this renderItems working? render intialCards", cardData)
       //console.log("is like and unlike here at ForEach?",cardData)
       //cardData.setLikeStatus(cardData.isLiked)
       //const cardElement = createCard(item)
      section.addItem(createCard(item));
      console.log("is this add Item firing", item)
     // section.renderItems(createCard(item));
+    //section.renderItems(cardData);
       //console.log("is like and unlike here at addItem???")
```

Comment:
`    section.renderItems(cardData.forEach((item) => {
`

This doesn't work

You need to make `renderItems` so that it could accept the array of cards